src/js/view/utils/skin.js

```diff
@@ -142,11 +142,14 @@ export function handleColorOverrides(playerId, skin) {
             addStyle([
                 // Toggle and menu button active colors
                 '.jw-button-color.jw-toggle:not(.jw-icon-cast)',
-                '.jw-button-color:hover:not(.jw-icon-cast)',
                 '.jw-button-color:focus:not(.jw-icon-cast)',
                 '.jw-button-color.jw-toggle.jw-off:hover:not(.jw-icon-cast)'
             ], 'color', config.iconsActive);
 
+            addStyle([
+                ':not(.jw-state-buffering) .jw-button-color:hover:not(.jw-icon-cast)'
```

Comment:
@DanFerrer would this prevent all icons from going to active state when hovered over ? I think the buffer icon is the only one that should be prevented

Reply:
@karimJWP Possibly, but the buffering icon is the only one that's shown when the player state is buffering. All other svgs with the class jw-button-color are hidden.

Reply:
@karimJWP - @DanFerrer is right in that it wouldn't. `:not(.jw-state-buffering)` ensures no hover color is applied when the player is in a buffering state.

Reply:
@karimJWP you're right, actually. Dan's addressing your feedback now.